src/app/add-category/add-category.page.ts

```diff
@@ -1,20 +1,22 @@
-import { Component, OnInit } from '@angular/core';
-import { Category } from '../session/session-state.service';
+import { Component } from '@angular/core';
+import { SessionState } from '../session/session-state.service';
+import { Router } from '@angular/router';
+import { Category } from '../session/session-state.typings';
+
 @Component({
   selector: 'app-add-category',
   templateUrl: './add-category.page.html',
   styleUrls: ['./add-category.page.scss'],
 })
-export class AddCategoryPage implements OnInit {
-
+export class AddCategoryPage {
   category: Category = new Category();
   icons: string[] = ['basket', 'cart', 'beer'];
-  constructor() { }
-
-  ngOnInit() {
-  }
+  constructor(private sessionState: SessionState, private router: Router) {}
```

Comment:
So here, we are bringing the sessionState in so that we can call addCategory, right? I feel like we didnt do this yesterday, just curious as to why we are doing it now.

Reply:
I probably just forgot to mention it. It is required to make calls to the service